index.html

```diff
@@ -1,259 +1,327 @@
-            <img src="images/Icons.png" alt="" />
+        <div class="about-skills">
+          <div class="skills about-framework">
+            <div class="icon">
+              <img src="images/Icons.png" alt="" />
+            </div>
+            <h4>Framework</h4>
+            <ul>
+              <li>Bootstrap</li>
+              <li>RSpec</li>
+              <li>Selenium</li>
+              <li>Ruby on Rails</li>
+              <li>Capybara</li>
+            </ul>
           </div>
-          <h4>Framework</h4>
-          <ul>
-            <li>Bootstrap</li>
-            <li>RSpec</li>
-            <li>Selenium</li>
-            <li>Ruby on Rails</li>
-            <li>Capybara</li>
-          </ul>
         </div>
-      </div>
-      <div class="about-skills">
-        <div class="skills about-skill">
-          <div class="icon">
-            <img src="images/Icons(1).png" alt="" />
+        <div class="about-skills">
+          <div class="skills about-skill">
+            <div class="icon">
+              <img src="images/Icons(1).png" alt="" />
+            </div>
+            <h4>Skills</h4>
+            <ul>
+              <li>CodeKit</li>
+              <li>GitHub</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
           </div>
-          <h4>Skills</h4>
-          <ul>
-            <li>CodeKit</li>
-            <li>GitHub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
+        </div>
         </div>
       </div>
-    </div>
-  </section>
-  <!-- Contact Section -->
-  <section class="contact-section">
-    <div class="container-contact">
-      <h2 class="contact-description">
-        I'm always interested in hearing about new projects, so if you'd like
-        to chat please get in touch.
-      </h2>
-      <form action="https://formspree.io/f/mnqrnjpr" method="POST">
-        <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" />
-        <input type="email" name="email" placeholder="Email address" required />
+    </section>
+    <!-- Contact Section -->
+    <section class="contact-section">
+      <div class="container-contact">
+        <div class="container-contact2">
+          <h2 class="contact-description">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mnqrnjpr" method="POST">
+          <div class="name-form">
+            <input
+            type="text"
+            name="firstName"
+            id="firstName"
+            placeholder="First Name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="text"
+            name="lastName"
+            id="lastName"
+            placeholder="Last Name"
+            maxlength="30"
+            required
+          />
```

Comment:
- [ ] Would you please use one input field for both `first` and `last` names as it is a project requirement

Reply:
- We have two inputs in the template that I choose in the desktop version
- They are in a div that is displayed none on the mobile version
![image](https://user-images.githubusercontent.com/11340240/186746144-ae90ea5a-134d-45a7-98ae-9578a7ff5ee1.png)